registry/tracecat_registry/integrations/ldap3.py

```diff
@@ -102,12 +108,22 @@ def find_users(self, base_dn: str, search_value: str):
     def get_user(self, user_dn: str) -> dict:
         return self._searchone(user_dn, "(objectClass=*)")
 
-    def disable_user(self, user_dn: str):
-        return self._connection.modify(user_dn, {"userAccountControl": [(2, [514])]})
+    def disable_user(self, user_dn: str) -> bool:
+        return self._modify(user_dn, "userAccountControl", [514])
 
-    def enable_user(self, user_dn: str):
-        return self._connection.modify(user_dn, {"userAccountControl": [(2, [512])]})
+    def enable_user(self, user_dn: str) -> bool:
+        return self._modify(user_dn, "userAccountControl", [512])
 
+    def set_account_expiration(self, user_dn: str, expiration_datetime = datetime.now()) -> bool:
```

Comment:
```suggestion
    def set_account_expiration(self, user_dn: str, expiration: datetime | None = None) -> bool:
      expiration = expiration or datetime.now(timezone.utc)
```

Reply:
Shouldn't set defaults dynamically in the method. This can be set in unexpected ways.